doc/tsc-meetings/2015-09-30.md

```diff
@@ -0,0 +1,161 @@
+Michael: The model is that we give the community time to catch up. In the future that’s 6 months, so we’re never going to be on a supported version.
+
+Ben: V8 moves quickly. If we stick with 4.5, we’re going to have to manage a bigger delta between our copy and upstream.
+
+James: Yes, this was always going to be the case. We’re committed to supporting the V8 for 36 months.
+
+Ben: Yes, my point is that we could be starting with a smaller delta.
+
+Rod: One of the reasons we had this LTS delay on 4 is to shake out some of these concerns, like the buffer changes. Normally this shakeout will be 6 months, but this time it’s only been a month.
+
+Alexis: Is v5 going to be LTS also?
+
+Rod: We are having discussions about the overlap between v5 and v6, whether there’ll be a strict cutoff or some overlap. No decision yet. That’ll happen in April. It sounds like most folks are leaning towards
+making the two releases (v4 LTS + v5) independent — option 1 [here](https://github.com/nodejs/node/issues/3000#issuecomment-144026295).
+
+Ben: We still haven’t come up with a way to maintain V8 going forward.
+
+James: We do need a good process there, you are absolutely right about that.
+
+Rod: I vote for IBM to hire someone dedicated to this. (Laughter)
+
+Michael: I can’t promise that at this point.
+
+Rod: As the delta increases between versions this becomes more concerning.
+
+James: make sure this is on the LTS WG agenda for next week.
+
+Rod: We tried to tackle this once before, came up with job description but nothing concrete.
+
+Michael: We stopped short of telling the foundation that this is a position that we need to staff.
+
+James: I would agree. We’re at the point where bringing someone in at the foundation level is probably a good idea.
+
+Rod: Noted. OK. We’ve got resolution there.
+
+Michael: James: Make sure it’s on the LTS agenda for going back to the foundation for resources.
+
+Rod: LTS is meeting this Monday (issue 43 on the repo). There’s some discussion around codenames and how to expose LTS info. We’ve committed to first week of October for LTS releases, so timing is going to be a little awkward with 4.1.2 coming out monday. Version 5 will be shipping when we have a stable V8, presumably mid-to-late october (1-3 weeks with no stable release line.)
+
+Rod: We’ll be pulling in semver minor and patch changes, we’ll continue to have bugfix releases for version 4,
+
+### Inspecting Node.js with Chrome DevTools [#2546](https://github.com/nodejs/node/issues/2546)
+
+Rod: Trevor is lead on this, but isn’t present. Does anyone else have any updates?
+
+Ben: Wasn’t the last contentious issue putting WS in core? I think we agreed that if it’s 
+
+Alexis: (discussion of another WS issue)
+
+Bert: we decided on this
+
+Ben: We agreed that if it was internal, non-exposed to core, 
+
+(Talk about native deps for WS, buffer-util can be )
+
+Ben: if this is a private dependency, we don’t need it to be fast, so we don’t necessarily need the native deps.
```

Comment:
"private debugger dependency"?  I remember it was in the context of not needing to be fast because it's only used in the debugger.